strings/remove_duplicate.py

```diff
@@ -1,14 +1,15 @@
-""" Created by sarathkaul on 14/11/19 """
-
-
 def remove_duplicates(sentence: str) -> str:
```

Comment:
We should rename the function to `remove_duplicate_whitespace()`

Reply:
Should I open another PR ?